pkg/persistence/config/internal/persistence/type_definition.go

```diff
@@ -227,7 +262,15 @@
 	case config.BucketType:
 		return BucketType, nil
 
-	default:
-		return nil, fmt.Errorf("unkown type: %T", c.Type)
+	case config.DocumentType:
```

Comment:
The language allows that, but to me, keeping "default" at the beginning or at the end helps to read the code easily.

Reply:
Here it would require me to duplicate the return, which I'd prefer not to